src/examples_mod.ts

```diff
@@ -0,0 +1,8 @@
+// Import some example files that aren't included in the dependency free, so we can compile them
+import "./_examples/main.ts"
+import "./_examples/components/visitor-demo.ts"
+import "./_examples/components/to-do/_to-do.ts"
+import "./_examples/components/array-demo.ts"
+import "./_examples/components/time-diff.ts"
+import "./_examples/components/async-demo.ts"
+import "./_examples/components/window-manager/_window-manager.ts"
```

Comment:
Note about this file that i want to mention:

If instead of setting the path to the js file inside the `tab-view.ts`, we imported the class and passed it in, we wouldn't need the `examples_mod.ts` file, and would only need to emit `_examples/main.ts`, so for example:

```ts
// src/_examples/components/tab-view.ts
import { AllComponents } from "all the files we need"

class TabView {
  #tabs = [
    {
      url: "/array-demo",
      component: ArrayDemo // before, it would be "./components/array-demo.js"
    }
  ]
}
```
Of course the hash router would need to account for this.

Curious on your thoughts @0kku?

Reply:
That would make it into a static import. It's intentionally dynamic: the component is not loaded until the user clicks on the tab. This makes initial load faster and avoids loading a potentially substantial number of pages that the user wasn't interested in visiting anyway. It's not a big deal on the example page because it's very small, but it's an example on how to do code splitting for a large website that could potentially have hundreds of large pages.

Reply:
very very good point 👍